strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/pypi/PypiBrowsePackageHtmlResponseBuilderTest.java

```diff
@@ -0,0 +1,143 @@
+package org.carlspring.strongbox.controllers.layout.pypi;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.carlspring.strongbox.artifact.coordinates.PypiArtifactCoordinates;
+import org.carlspring.strongbox.config.IntegrationTest;
+import org.carlspring.strongbox.providers.io.RepositoryFiles;
+import org.carlspring.strongbox.providers.io.RepositoryPath;
+import org.carlspring.strongbox.storage.repository.Repository;
+import org.carlspring.strongbox.testing.artifact.ArtifactManagementTestExecutionListener;
+import org.carlspring.strongbox.testing.artifact.PypiTestArtifact;
+import org.carlspring.strongbox.testing.repository.PypiTestRepository;
+import org.carlspring.strongbox.testing.storage.repository.RepositoryManagementTestExecutionListener;
+
+import javax.inject.Inject;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+/**
+ * @author ankit.tomar
+ */
+@IntegrationTest
+public class PypiBrowsePackageHtmlResponseBuilderTest
+{
+
+    private static final String REPOSITORY_RELEASES = "pypi-releases-test";
+
+    private static final String STORAGE_PYPI = "storage-pypi-test";
+
+    @Inject
+    private PypiBrowsePackageHtmlResponseBuilder htmlResponseBuilder;
+
+    @Test
+    @ExtendWith({ RepositoryManagementTestExecutionListener.class })
+    public void testNoPackageFoundFlow(@PypiTestRepository(repositoryId = REPOSITORY_RELEASES,
+                                                           storageId = STORAGE_PYPI) 
+                                       Repository repository)
```

Comment:
@anki2189 don't need the `Repository`  here, as we expect that there will be no `Reository` parameter for `PypiBrowsePackageHtmlResponseBuilder#getHtmlResponse` method.

Reply:
Fixed.